ptu/global.ini

```
@@ -26917,11 +26917,11 @@ PU_EDL1_FO_FuelRepairRestock_IG_001_StopByThe=Schauen Sie im Hauptbüro vorbei,
 PU_EDL1_FO_LandingBayAssigned_IG_001_LandingPadAssigned=Landeplattform zugewiesen.
 PU_EDL1_FO_LandingComplete_IG_001_LandingCompleteHave=Landung abgeschlossen. Hab einen angenehmen Aufenthalt.
 PU_EDL1_FO_Paid_IG_001_PaymentaAccepted=Zahlung akzeptiert.
-PU_EDL1_FO_ProceedToLandingBay_IG_001_PleaseProceedTo=Bitte begeben Sie sich zum zugewiesenen Landeplatz.
+PU_EDL1_FO_ProceedToLandingBay_IG_001_PleaseProceedTo=Bitte begeben Sie sich zur zugewiesenen Landeplattform.
 PU_EDL1_FO_RevokeLanding_IG_001_DueToRepeated=Aufgrund wiederholter Verstöße gegen Hangar-Sicherheitsprotokolle wurden Ihre Landerechte widerrufen.
 PU_EDL1_FO_RevokeLanding_IG_002_YourLandingPrivileges=Ihre Landeprivilegien wurden widerrufen.
-PU_EDL1_FO_ShipRequests_Delivered_IG_001_YourShipHas=Ihr Schiff wurde an den Landeplatz geliefert.
-PU_EDL1_FO_ShipRequests_NoAvailableBays_IG_001_ImSorryWe=Es tut mir leid, wir haben keine verfügbaren Landeplätze. Ihre Anfrage wurde bearbeitet und Ihr Schiff wird im nächsten verfügbaren Hangar abgeliefert.
+PU_EDL1_FO_ShipRequests_Delivered_IG_001_YourShipHas=Ihr Schiff wurde an die Landeplattform geliefert.
+PU_EDL1_FO_ShipRequests_NoAvailableBays_IG_001_ImSorryWe=Es tut mir leid, wir haben keine verfügbaren Landeplattformen. Ihre Anfrage wurde bearbeitet und Ihr Schiff wird im nächsten verfügbaren Hangar abgeliefert.
```

Fände ich schöner... gibt noch mehr Vorkommen
```suggestion
PU_EDL1_FO_ShipRequests_NoAvailableBays_IG_001_ImSorryWe=Es tut mir leid, wir haben keine verfügbaren Landeplattformen. Ihre Anfrage wurde bearbeitet und Ihr Schiff wird in den nächsten verfügbaren Hangar geliefert.
```

---

Wo denn überall?